docs/cli/coder_config-ssh.md

```diff
@@ -42,6 +42,13 @@ Specifies the path to an SSH config.
 | Consumes | <code>$CODER_SSH_CONFIG_FILE</code> |
 | Default | <code>~/.ssh/config</code> |
 
+### --ssh-host-prefix
+
+Override the default host prefix.
+<br/>
+| | |
+| --- | --- |
```

Comment:
Hmm, is this a bug in gen?

Reply:
Uh,I guess if the default is `""`, it doesn't put anything?